### PR TITLE
Crew console shows if someone is DNR

### PR DESCRIPTION
**Pull request by E231kei500bandai**
## About The Pull Request
Based on Skyrat-SS13/Skyrat-tg/pull/19629
This is the second Crew Monitor PR that I took from Skyrat

Adds a new entry to the table "is_revivable"  based on the generate_death_examine_text proc
Someone is considered not revivable if they are disconnected, brainless, or DNR

If someone is dead and DNR, their icon set to a no symbol
Dead people with binary sensors get "Dead" replaced with "Dead (DNR)"
Theoretically, the health value information for vital sensors could still be useful so I left that

Example:
image

Some things that I was unsure if they should be shown on the crew monitor or not:

Blacklisted from revival
No brain
Disconnected from the server

## Why It's Good For The Game

It's easier to filter out the bodies that don't matter
## Changelog
:cl:
qol: Crew Monitor now displays when someone is DNR
/:cl:

### CODE REVIEW

**Comment by ArcaneMusic** [comment]
Is there a dedicated icon for critical condition? Or are the icons mostly for suit sensor status?

File: code/game/machinery/computer/crew.dm [hunk 1]
```diff
@@ -246,6 +248,14 @@ GLOBAL_DATUM_INIT(crewmonitor, /datum/crewmonitor, new)
 		if (sensor_mode >= SENSOR_LIVING)
 			entry["life_status"] = (tracked_living_mob.stat != DEAD)
 
+		// Not revivable if no key, no brain, or DNR
+		if (sensor_mode >= SENSOR_LIVING)
+			var/mob/dead/observer/ghost = tracked_living_mob.get_ghost(TRUE, TRUE)
```
Comment (Mothblocks) [inline code/game/machinery/computer/crew.dm:253]: Named parameters

**Comment by Helg2** [comment]
How this works with just beheaded people? As i remember they get 💀icon (in med hud) and it will look quite problematic if someone is cutting heads off.

**Comment by optimumtact** [comment]
agree with melbert, this makes suit sensors even better, we're trying to avoid them being too good and having people be unrevivable that someone has to go and verify is one of the downsides.

perhaps allow people to mark others unrevivable and record that status in the crew monitor instead?

**Comment by BlueMemesauce** [comment]
probably not going to do that

**Pull request closed without merging**